XX_Jenkins_intro/README.MD

```diff
@@ -0,0 +1,17 @@
+# Alliedium DevOps course 2022
+
+# Jenkins part 1
+## Prerequisites for your local machine
+1. Ubuntu linux. You can follow instructions from the [link](https://github.com/Alliedium/awesome-ansible#2-general-notes-on-creating-your-own-custom-inventory-and-playbooks) to install Ubuntu linux
+2. Ansible. Please follow the step 2 of the [instructions](https://github.com/Alliedium/awesome-ansible#2-general-notes-on-creating-your-own-custom-inventory-and-playbooks)
+3. Install openJdk-17
+   `sudo apt update`
+   `sudo apt upgrade`
+   `sudo apt install -y openjdk-17-jdk`
```

Comment:
What is the reason to install Java on local machine? And what this "local machine" means? This is misleading. And what if Java is already installed, but for different version (or versions)? How to avoid conflicts between different versions of Java?

Reply:
Previously we discussed that Jenkins will use installation from local machine/vm. So "local machine" means the machine where Jenkins will be installed.  The java 17 version is the requirement of the spring-boot-api application.
 I agree that the section how to avoid conflicts should be added